src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
-public class MainApp {
-
+public abstract class MainApp {
+    public static void main(String [] args) {
+        Machine [] machines = {new Truck(),new Bulldozer(),new Excavator()};
```

Comment:
add spaces here, use IDE's autoformatter, Ctrl Alt L for windows machines

Reply:
Thanks, spaces fixed